src/libsyncengine/jobs/network/csvfullfilelistwithcursorjob.cpp

```diff
@@ -63,6 +63,7 @@ bool CsvFullFileListWithCursorJob::getItem(SnapshotItem &item, bool &error, bool
     bool prevCharDoubleQuotes = false;
     bool readNextLine = true;
     std::string tmp;
+    uint doubleQuoteCounter = 0;
```

Comment:
```suggestion
    uint doubleQuoteCount = 0;
```

Simply because it is shorter to type!

Reply:
Ok, done 👍